HW0/P6/P6B.py

```diff
@@ -16,16 +17,27 @@ def burnTime(t):
 
     # Use a variety of wait times
     ratio = []
-    wait_time = []
+    wait_time = [10**(-i) for i in xrange(7)]
 
     for t in wait_time:
         # Compute jobs serially and in parallel
         # Use time.time() to compute the elapsed time for each
-        serialTime = 1
-        parallelTime = 1
+
+        # Serial
+        start = time.time()
+        pool.map(burnTime(t), range(N))
```

Comment:
The use of pool.map is indeed incorrect. If you look at the graph, you can see that you get a speedup that is higher than 4 (the number of processes).